examples/soundcloud.amp.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Soundcloud examples</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <script async custom-element="amp-soundcloud" src="/dist/v0/amp-soundcloud-0.1.max.js"></script>
```

Comment:
remove the .max here and you should be able to do dev work.

when this is ready for merge, make sure to switch it to the cdn path.